### DISCUSSION TITLE
How can the UART of the JR interface be connected to the ESP32 S3 so that it can use the PPM input normally without the backpack interfering with each other?

### DISCUSSION THREAD

**Discussion by XH-FPV** [ExpressLRS/ExpressLRS#2720] [Q&A]
I encountered a problem when DIY Gemini transmitter: the UART0 of the ESP32 S3 is connected to the backpack, if the UART of the JR interface is also connected to the UART0 of the ESP32 S3, it will cause them to interfere with each other and cannot communicate normally, how should I connect them?
![Screenshot_20240521-085500_EDIT_1716252931062_EDIT_1716252957932](https://github.com/ExpressLRS/ExpressLRS/assets/134117214/7983a584-4032-4a1c-b2e0-baae6138e4b1)

**Comment by pkendall64** [comment]
Connect you PPM signal to the SPORT pin (GPIO11) and it will auto-detect.
The PPM feature is for radios that do not support CRSF protocol.

**Reply by XH-FPV** [reply]
Isn't it like TBS Crossfire?

**Reply by pkendall64** [reply]
An external module uses the SPORT pin to communicate with the handset using the CRSF protocol.
In the JSON file you need to define serial_rx and serial_tx to be 11 which is the SPORT GPIO pin.
